src/test/java/annotatedspring/episodes/EpisodeTest.java

```diff
@@ -31,4 +33,26 @@ public void trim_episode_title() {
 
         assertThat(episode.getTitle(), is(equalTo("no leading nor trailing spaces")));
     }
+    
+    @Test
+    public void markdown_table() {
+        Episode episode = new Episode();
+        String notes = "| Name| Value|\r\n"
+                + "| ---- | ----- |\r\n"
+                + "| Hello| World!|\r\n"
+                + "| Another| Row|\r\n";
+        
+        episode.setNotes(notes);
+        
+        assertThat(episode.getNotesHtml(), containsString("<table>"));
+        assertThat(episode.getNotesHtml(), containsString("<th>Name</th>"));
+        assertThat(episode.getNotesHtml(), containsString("<th>Value</th>"));
+        assertThat(episode.getNotesHtml(), containsString("<tbody>"));
+        assertThat(episode.getNotesHtml(), containsString("<tr>"));
+        assertThat(episode.getNotesHtml(), containsString("<td>Hello</td>"));
+        assertThat(episode.getNotesHtml(), containsString("<td>World!</td>"));
+        assertThat(episode.getNotesHtml(), containsString("</tr>"));
+        assertThat(episode.getNotesHtml(), containsString("</table>"));
+        System.out.println(episode.getNotesHtml());
```

Comment:
Can you please remove this? I'm guessing it was there for debugging purpose.

Also, I'd be fine with the test just making sure there's a `<table>` in there and not worry about the individual headers and cells.